config/database.yml

```diff
@@ -5,16 +5,38 @@ default: &default
   timeout: 5000
 
 development:
-  <<: *default
-  database: cfe_civil_development
+  primary:
+    <<: *default
+    database: cfe_civil_development
+  request_log:
+    <<: *default
+    database_tasks: false
+    host: localhost
+    username: <%= ENV['POSTGRES_USER'] %>
+    password: <%= ENV['POSTGRES_PASSWORD'] %>
+    database: <%= ENV['POSTGRES_DATABASE'] %>
+    port: <%= ENV['POSTGRES_PORT'] %>
 
 test:
-  <<: *default
-  database: cfe_civil_test
+  primary:
+    <<: *default
+    database: cfe_civil_test
+  request_log:
+    <<: *default
+    database_tasks: false
+    database: cfe_civil_test
 
 production:
-  <<: *default
-  host: <%= ENV['POSTGRES_HOST'] %>
-  username: <%= ENV['POSTGRES_USER'] %>
-  password: <%= ENV['POSTGRES_PASSWORD'] %>
-  database: <%= ENV['POSTGRES_DATABASE'] %>
+  primary:
+    <<: *default
+    host: <%= ENV['POSTGRES_HOST'] %>
+    username: <%= ENV['POSTGRES_USER'] %>
+    password: <%= ENV['POSTGRES_PASSWORD'] %>
+    database: <%= ENV['POSTGRES_DATABASE'] %>
+  request_log:
```

Comment:
It seems really strange having a "primary" database and a "request_log" database. Add a comment explaining?

How about just having this database config in the 'development' environment? Or even creating a separate environment 'remote_database' just for this purpose?

Reply:
I had that exact thought (the remote database env) but couldn't decide if it was a good idea or not